.npmignore

```diff
@@ -0,0 +1,3 @@
+/.*
+/.*/
```

Comment:
Dot-files are all configs. Dot-dirs are often used by editors/ide, like
'.idea'. I usually prefer to add them just in case.